src/main/java/it/pagopa/transactions/commands/handlers/TransactionInizializeHandler.java

```diff
@@ -140,7 +151,13 @@ public Mono<NewTransactionResponseDto> handle(TransactionInitializeCommand comma
                             activatePaymentNoticeRes.getPaymentToken());
                     return transactionEventStoreRepository.save(transactionInitializedEvent)
                             .thenReturn(transactionInitializedEvent);
+                        })
+                .doOnNext(transactionInitializedEvent -> {
+                    queueAsyncClient.sendMessageWithResponse(
+                            BinaryData.fromObject(transactionInitializedEvent),
+                            Duration.ofSeconds(Integer.valueOf(queueVisibilityTimeout)), null);
                 })
+                .doOnError(throwable -> log.error("Failed sending transactionInitializedEvent queue", throwable))
```

Comment:
I think [`onErrorContinue`](https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Mono.html#onErrorContinue-java.util.function.BiConsumer-) is the right combinator to use when catching errors from `doOnNext`